setup.py

```diff
@@ -13,6 +13,7 @@
     description='Human Cell Atlas Data Storage System Command Line Interface',
     long_description=open('README.rst').read(),
     install_requires=[
+        "requests==2.17.3"
         'boto3 >= 1.4.4, < 2'
```

Comment:
This is a syntax error, isn't it? Also, please remove boto3 if we're not going to use it.